Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -310,6 +310,18 @@ extension InternalRouter where Self: Router {
         routeTask?.cancel()
         let options = progress.reroutingOptions(with: origin)
         
+        if isRerouting {
+            switch (delegate?.router(self, maneuverOffsetWhenReroutingFrom: origin) ?? RouteController.DefaultBehavior.reroutingManeuverRadius) {
+            case .disabled:
+                options.initialManeuverAvoidanceRadius = nil
```

Comment:
This is `nil` when talking to the Directions API because the Directions API’s default behavior is not to avoid a maneuver at all. But since the default behavior in the navigation SDK is different, `nil` can have a different meaning.

Reply:
Default behavior is configured by `RouteOptions.initialManeuverAvoidanceRadius`. `ReroutingManeuverOffset` is used then to avoid using `nil`s and thus remove the ambiguity of it's meaning by asking directly if user wants the `default` is he configured previously or to override the value. If user didn't configure `initialManeuverAvoidanceRadius` and left it too as a default - this is the same default as Directions expect.